public/js/index.js

```diff
@@ -200,65 +46,15 @@ jQuery(function () {
     document.getElementById('mainImage').src = `data:image/jpeg;base64,${strImage}`
   })
 
+  return objSocket
+}
+
+const initMisc = function (objSocket) {
```

Comment:
This is a really poor function name, especially since it doesn't have a standard description preceding it. Please assign a descriptive name and add a description comment.